webpack.config.js

```diff
@@ -0,0 +1,52 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const HtmlWebpackInlineSVGPlugin = require('html-webpack-inline-svg-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: ['./src/index.js', './src/styles/main.css'],
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+  devServer: {
+    contentBase: path.join(__dirname, 'dist'),
+    port: 9000,
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(css|scss)$/,
+        include: path.resolve(__dirname, 'src/styles'),
+        use: ['style-loader', 'css-loader', 'sass-loader'],
+      },
+      { test: /\.html$/, use: 'html-loader' },
+      {
+        test: /\.(png|ttf)$/,
```

Comment:
according to this rule, the `ttf` fonts will be put to `images` folder, which is confusing
Please create a separate rule for `ttf`